include/openthread/channel_manager.h

```diff
@@ -47,8 +47,12 @@ extern "C" {
  * @brief
  *   This module includes functions for Channel Manager.
  *
- *   The functions in this module are available when Channel Manager feature
- *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) is enabled. Channel Manager is available only on an FTD build.
+ *   The functions in this module are available when Channel Manager features
+ *   (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE`) and (`OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`)
```

Comment:
You can remove the parentheses ( ) around all these config variables, they are needed